### ISSUE TITLE
HD Text Persists Display on top level

### ISSUE THREAD

**Issue by wu-kang66** [bayaraa/d2gl#141]
![20240207160344](https://github.com/bayaraa/d2gl/assets/157418566/cc7b464b-1afb-42a6-a388-605b9151c7ee)
![20240207160355](https://github.com/bayaraa/d2gl/assets/157418566/c61832bd-8067-459e-b9ca-81a80e78a44f)
![20240207160401](https://github.com/bayaraa/d2gl/assets/157418566/fd231deb-41c2-4fb4-a2df-f5de32f28f89)

**Comment by bayaraa** [comment]
Yeah this is known. If message is not important just remove it on your end.

**Comment by wu-kang66** [comment]
thanks for answer@bayaraa .will it be fixed in the next version?

**Comment by bayaraa** [comment]
I mean if its not important remove on your end (remove text or set blank text on your server side). Recently I dont have time to work on d2gl. sorry.

**Comment by wu-kang66** [comment]
ok
